### PR TITLE
Fix relative timestamp and alignment in deleted items list

### LINKED ISSUE #684: wrong timestamp is shown in timeline
**Describe the bug**
When i try to undeleted a card from a shared board and i have no right to share, edit and manage the time of deletion is wrong

**To Reproduce**
1. share a board to user 2 and share without rights
2. delete a card from the shared board
3. as user 2 try to undelete the card
4. See "49 years ago"

**Expected behavior**
The correct time is shown or don't show the undelete icon to user 2

**Screenshots**
screenshot_2018-11-09 testboard deck - nextcloud
screenshot_2018-11-09 testboard deck - nextcloud 1
screenshot_2018-11-09 testboard deck - nextcloud 2

### PR DESCRIPTION
**Pull request by juliushaertl**
Fixes #684

image

### CODE REVIEW

File: templates/part.board.sidebarView.php [hunk 1]
```diff
@@ -122,33 +122,30 @@
...
-        	  <a ng-click="cardOrCardAndStackUndoDelete(deletedCard)">
-              <span class="icon icon-history"></span>
-            </a>
-    	   </li>
-    	</ul>
-    </div>
+			<li class='board-detail__deleted-list__item' ng-repeat="deletedCard in cardservice.deleted">
+				<span class="icon icon-deck"></span>
+				<span class="title">{{deletedCard.title}} ({{stackservice.tryAllThenDeleted(deletedCard.stackId).title}})</span>
+				<span class="live-relative-timestamp" data-timestamp="{{ deletedCard.deletedAt*1000  }}">{{deletedCard.deletedAt | relativeDateFilter }}</span>
```
Comment (LemonAndroid) [inline templates/part.board.sidebarView.php]: Is the `live-relative-timestamp` functionality new?

Comment (juliushaertl) [reply templates/part.board.sidebarView.php]: It is a class the Nextcloud server uses to update the relative timestamp regularly.

Comment (LemonAndroid) [reply templates/part.board.sidebarView.php]: Was there any mistake with the previous implementation except for not using this?

Comment (juliushaertl) [reply templates/part.board.sidebarView.php]: All fine. :wink:

**Pull request merged**